api/src/main/java/io/druid/segment/loading/DataSegmentPusherUtil.java

```diff
@@ -54,14 +54,25 @@ public static String getStorageDir(DataSegment segment)
   public static String getHdfsStorageDir(DataSegment segment)
```

Comment:
Same as above, changing this function might break custom extensions using this function.